### PR TITLE
add assertIndexesAndConstraints on OGM

### PR DESCRIPTION
**Pull request by g10**
# Description

`assertIndexesAndConstraints` is not available on the OGM class, therefore it's required to add the `@neo4j/graphql` package (in addition to `@neo4j/graphql-ogm`) to add the indexes and constraints specified in the schema

more details → https://github.com/neo4j/graphql/issues/3116

## Complexity

Complexity: Low

# Issue

Closes https://github.com/neo4j/graphql/issues/3116 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [x] Documentation has been updated
- [x] TCK tests have been updated
- [x] Integration tests have been updated
- [ ] Example applications have been updated
- [x] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by darrellwarde** [comment]
Hey @g10, thanks for the contribution! A few things:

- Thanks for sending the email signing the CLA. We're also trialling a new system at the moment, so if you would mind clicking the button in the comment from @CLAassistant and signing there also, it would be appreciated.
- Please add a changeset by running `yarn changeset` in the root of the repo, following the instructions and adding an appropriate changelog message.
- If possible, please add at least one integration test for this functionality. I realise that it is tested in the library itself and the OGM isn't all that well tested, but we need to be aiming to improve test coverage here.

Thanks again, I look forward to seeing this progress!

**Comment by g10** [comment]
hi @darrellwarde, added the change requests, let me know if this works

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-3117.surge.sh

**Review by darrellwarde — CHANGES_REQUESTED** [review]
I'm so sorry about the slow review on this one - I would like to see the new code tweaked a bit, but the feature is a nice addition!

File: packages/ogm/tests/integration/neo4j.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ async function connect(): Promise<neo4j.Driver> {
         return driver;
     }
 
-    const { NEO_USER = "neo4j", NEO_PASSWORD = "password", NEO_URL = "neo4j://localhost:7687/neo4j" } = process.env;
+    const { NEO_USER = "neo4j", NEO_PASSWORD = "NeedsALongPassword", NEO_URL = "neo4j://localhost:7687/neo4j" } = process.env;
```
[the remark this answers is not in the document]
Comment (g10) [reply packages/ogm/tests/integration/neo4j.ts:30]: true, guess I've aligned it with the one in [`docker-compose.yaml`](https://github.com/neo4j/graphql/blob/dev/docker-compose.yaml) but there's no need for that as it is the default 😁

**Review by darrellwarde — APPROVED** [review]

**Pull request merged**